lib/experimental-default-theme.json

```diff
@@ -131,6 +131,42 @@
 						"slug": "huge",
 						"value": 48
 					}
+				],
+				"fontFamilies": [
+					{
+						"value": "-apple-system,BlinkMacSystemFont,\"Segoe UI\",Roboto,Oxygen-Sans,Ubuntu,Cantarell,\"Helvetica Neue\",sans-serif",
+						"slug": "system",
+						"name": "System",
+						"group": "web-safe"
+					},
+					{
+						"value": "Helvetica Neue, Helvetica, Arial, sans-serif",
+						"slug": "helvetica-arial",
+						"group": "web-safe"
+					},
+					{
+						"value": "Geneva, Tahoma, Verdana, sans-serif",
+						"slug": "geneva-verdana",
+						"group": "web-safe"
+					},
+					{
+						"value": "Cambria, Georgia, serif",
+						"slug": "cambria-georgia",
+						"group": "web-safe"
+					},
+					{
+						"value": "Hoefler Text, Baskerville Old Face, Garamond, Times New Roman, serif",
+						"slug": "hoefler-times-new-roman",
+						"group": "web-safe"
+					},
+					{
+						"value": "Consolas, monaco, monospace",
+						"slug": "consolas",
+						"group": "web-safe"
+					}
+				],
+				"fontFamilyGroups": [
```

Comment:
What do we need `fontFamilyGroups` for?

Reply:
> What do we need `fontFamilyGroups` for?

I assume it's for this: https://github.com/WordPress/gutenberg/pull/24868#issuecomment-686557935

Reply:
I'm continuing this conversation at https://github.com/WordPress/gutenberg/pull/24868#discussion_r492057539